web/src/UserMangaRanking.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './App.css';
+
+export class UserMangaRanking extends Component {
+  render() {
+    return (
+      <div className="User-manga">
+        <span className="User-completed">Completed</span> mangas gave&nbsp;
+        sweetmonia <span className="User-rancompleted">#857</span> place in
+        ranking, which is <span className="User-days">795</span>&nbsp;chapters
+        and <span className="User-days">631</span> volumes read. sweetmonia
+        didn't finished enough mangas that is&nbsp;
```

Comment:
"didn't finish", and "manga" (the same way you don't use "animes")

Reply:
also done :)